pfcpiface/parse-far_test.go

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2022-present Open Networking Foundation
+
+package main
+
+import (
+	"net"
+	"testing"
+
+	"github.com/omec-project/upf-epc/test/integration"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/wmnsk/go-pfcp/ie"
+)
+
+type testCase struct {
+	input       *ie.IE
+	op          operation
+	description string
+}
+
+func TestParseFAR(t *testing.T) {
+	createOp, updateOp := create, update
+
+	for _, scenario := range []testCase{
+		{
+			op:          createOp,
+			input:       integration.NewUplinkFAR(integration.IEMethod(createOp), 999, ActionDrop),
```

Comment:
we should use pfcpsim package instead of `integration`

Reply:
I opted on using IE library directly. 

Importing `pfcpsim` gives me an error (omitting path and sha): `Build constraints excludes all go files in [...]/pfcpsim@[...]`